hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/directory_markers.md

```diff
@@ -306,7 +290,6 @@ line of bucket policies via the `-marker` option
 
```

Comment:
Should we edit this table as well ? delete and authoritative are not valid anymore